shell/content/docs/en-us/whats-new.md

```diff
@@ -2,6 +2,33 @@
 title: What's New in 2.6
 ---
 
+
+## New in 2.6.7
+
+### Amazon Billing in the Marketplace
+
+- Now you can purchase support for Rancher through the AWS marketplace 
+
+### Ability to configure Azure AD with MSAL
+
+- [need description]
+
+### Kubernetes 1.24
+
+- Kubernetes 1.24 is now available as a supported Kubernetes version option when provisioning RKE1 clusters as well as upgrading imported RKE2/k3s clusters.
```

Comment:
We support running Rancher itself on 1.24. We also support it for downstream provisioned RKE2 clusters (not just imported ones).

So, I think we should state something like this:
```suggestion
- Kubernetes 1.24 is now supported Kubernetes version for running Rancher as well for provisioned and imported RKE1/RKE2/k3s clusters.
```

@jiaqiluo @kinarashah @sowmyav27 @slickwarren, please chime in if the above is incorrect.

Reply:
The proposed changes LGTM.  
A side note is that now 1.24 is the default version when creating an RKE1/RKE2/K3s cluster via Rancher UI and the default version for RKE CLI.

Reply:
I agree, LGTM